### ISSUE TITLE
Maintenance plan draft - request feedback on *requirements*

### ISSUE THREAD

**Issue by briesenberg07** [pan-zhuo/OMR#1]
@uwlib-cams/uwlswd
@pan-zhuo 
@AdamSchiff
@CECSpecialistI
@gerontakos
@JianPLee
@junghaelee

Following up re: a maintenance plan for the [the MARC21 Vocabularies](http://www.marc21rdf.info/), which we are planning to publish as UW Libraries Semantic Web Data (UWLSWD), as discussed [2023-01-24](https://docs.google.com/document/d/13OK-OqBnqaihnfByu-wZa65F0AlpVTrUYHWZs2SZAUk/edit#bookmark=id.5nodlfu6s5kb)\*\*.

Please provide feedback on the document linked below. It isn't really a maintenance plan (yet). At this point I want to make sure that we gather all the relevant requirements for maintenance. I thought of two (2) such requirements. But are there others?

> [DRAFT_maintenance_plan_resources](https://docs.google.com/document/d/127vvBwnYHO2RVpZzWlf-pE418V1BwFRMUqizU_1weFA/edit?usp=sharing)\*\*

\*\* *UW NetID access only*

**Comment by briesenberg07** [comment]
Thank you for your feedback on the requirements for maintenance @CECSpecialistI @pan-zhuo and @AdamSchiff. I've drafted a brief high-level maintenance plan based on the requirements we outlined.

> resources_DRAFT_maintenance_plan \> [DRAFT maintenance plan](https://docs.google.com/document/d/127vvBwnYHO2RVpZzWlf-pE418V1BwFRMUqizU_1weFA/edit#bookmark=id.jm7wucd64pjr)

The last paragraph sort of punts because it refers to doing things (like versioning the vocabularies) that are complex in their own right and for which we don't yet have procedures. But we have said that we'll work on versioning procedures and [are beginning that discussion](https://github.com/orgs/uwlib-cams/teams/uwlswd/discussions/6).

CC additional LDT members @gerontakos @JianPLee @junghaelee
